NeuralAmpModeler/config.h

```diff
@@ -56,8 +56,16 @@
 
 #define ROBOTO_FN "Roboto-Regular.ttf"
 #define HELP_FN "help.svg"
-#define FILE_FN "file.svg"
+#define FILE_FN "SkinEHeritage_File.svg"
 #define FOLDER_FN "folder.svg"
-#define CLOSE_BUTTON_FN "close-button.svg"
+#define CLOSE_BUTTON_FN "SkinEHeritage_Close-button.svg"
```

Comment:
Are those old SVGs used anywhere else? If not, they can be removed as well.

Reply:
you mean the actual .svg files inr /img folder ?
they shouldnt be needed anymore, right...